CONTRIBUTE.md

```diff
@@ -1,84 +1,93 @@
+3. Clone your fork to your local computer or cloud workspace using your preferred command line interface after navigating to the directory you want to place the repository in:
 
     ```
     git clone your-fork-repository-url
     ```
 
-    - Change directories
+    - Change directories to the one you cloned
 
     ```
     cd repository-name
     ```
 
-    - Add the upstream repository
+    - Add the upstream repository, this is the original repository that you want to contribute to.
 
     ```
     git remote add upstream original-repository-url
     ```
 
-    - Using `git remote -v` will show two remote repositories named:  
-        - `upstream`, which refers to the repository  
-        - `origin`, which refers to your personal fork  
-    - Often, updates to an `upstream` repository will occur while you are developing changes on your personal fork. You can pull the latest changes from `upstream`, including tags:
+    - You can use the following to view the remote repositories:
 
     ```
-    git checkout dev
-    git pull upstream dev --tags
+    git remote -v
     ```
 
+    - `upstream`, which refers to the original repository  
+    - `origin`, which refers to your personal fork  
+
 4. Develop your contribution:
     - Create a new branch named appropriately for the feature you want to work on:
 
     ```
-    git checkout -b feature-name
+    git checkout -b new-branch-name
+    ```
+
+    - Often, updates to an `upstream` repository will occur while you are developing changes on your personal fork. You can pull the latest changes from `upstream`
+
+    ```
+    git pull upstream dev
+    ```
+
+    - Commit locally as you progress using `git add` and `git commit`.` For example, updating a readme.md file:
+
+    ```
+    git add readme.md
+    git commit -m "updated readme file"
     ```
 
-    - Commit locally as you progress using `git add` and `git commit`. You can check the status of your local copy of the repository using
+    - You can check the status of your local copy of the repository to see what pending changes have not been added or committed using:
 
     ```
+    git status
     ```
 
-5. To submit your contribution, push your changes back to your fork on Github:
+    - After making some changes, push your changes back to your fork on GitHub:
 
     ```git
     git push origin feature-name
     ```
 
     - Enter username and password, depending on your settings, you may need to use a [Personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
-    - Make a Pull Request. On the GitHub page for your personal fork, there will be a green Pull Request button.    -
 
-## Adding New Notebooks or Example Workflows
+5. To submit your contribution, make a pull request to the `dev` branch our repository on the GitHub page. There will be a green Pull Request button.  
```

Comment:
@ebolch, Submit to the 'dev' branch? Are we planning to maintain two identical branches?